### PR TITLE
Dra 437 enable oauth

### PR DESCRIPTION
**Pull request by thomasegense**
OAuth enabled on module.
No method defined to use Oauth yet.

### CODE REVIEW

File: src/main/openapi/ds-license-openapi_v1.yaml [hunk 1]
```diff
@@ -523,3 +567,30 @@ components:
...
+  # TODO: Inject this from properties when DRA-139 has been implemented
+  securitySchemes:
+
+    KBOAuth:
+      type: oauth2
+      description: 'This API uses OAuth 2 with the implicit grant flow. [More info](https://api.example.com/docs/auth)'
+      flows:
+        implicit:   # <---- OAuth flow(authorizationCode, implicit, password or clientCredentials)
+          authorizationUrl: 'https://keycloak-devel-01.kb.dk/realms/ds-poc/protocol/openid-connect/auth'
+          #authorizationUrl: https://api.example.com/oauth2/authorize # Overridden by the configuration
```
Comment (VictorHarbo) [inline src/main/openapi/ds-license-openapi_v1.yaml:582]: As the other services. Please replace with an injection of the url.

**Review by VictorHarbo — CHANGES_REQUESTED** [review]
Looks good. I think you need to add config files to aegis for this one. Also, there are three integration tests failing. However, they are also failing on master and dont seem to be related to this PR. If you won't fix them here, could you make a JIRA for them? Also one code comment.

Comment: 
1)aegis file added now.
2) Jira created for integration test issues.

File: src/main/openapi/ds-license-openapi_v1.yaml (hunk 1 above)
Comment (thomasegense) [reply src/main/openapi/ds-license-openapi_v1.yaml:582]: Replaced with comment.

**Review by VictorHarbo — APPROVED** [review]
Yes. LGTM!

**Pull request merged**